spec/factories/notifications.rb

```diff
@@ -3,6 +3,7 @@
     notifiable { |notification| notification.association :issue }
     association :actor, factory: :user
     association :recipient, factory: :user
+    read_at :nil
```

Comment:
Watch out - this says `:nil` (i.e. a symbol), not `nil`.

But `read_at nil` is redundant anyway - you can leave the entire line out and FactoryBot will set it to `nil` by default